indico/htdocs/js/indico/modules/static_filters.js

```diff
@@ -77,18 +74,19 @@
         // Needed because $(window).scroll() is not called when hiding elements
         // causing scrolling elements to be out of place.
         $(window).trigger('scroll');
-    };
+    }
 
     global.setupSearchBox = function setupSearchBox(config) {
-        searchBoxConfig = config;
+        var applySearchFilters = _.partial(_applySearchFilters, config);
```

Comment:
`_applySearchFilters.bind(config)`  should do the same (we don't support any browsers that do not support `<function>.bind()` anymore)